test/e2e/spec/eid_service/tcToken_spec.js

```diff
@@ -1,50 +1,45 @@
 describe("eID-Service: The tcToken generator", function() {
 
-    var frisby = require('frisby');
+    var chakram = require('/usr/local/lib/node_modules/chakram/lib/chakram.js'),
+	expect = chakram.expect;
     var URL = 'https://' + process.env.BOILERPLATE_DOMAIN + '/api/eIdService/getTcToken';
 
-    beforeAll(function() {
-        frisby.addExpectHandler('tc token', function(response) {
-            let xml = response._body.replace(/\n/g, "").replace(/\r/g, "").replace(/ /g, "");
-
-            expect(xml).toEqual(jasmine.stringMatching(/^<TCTokenType><ServerAddress>.*<\/Binding><\/TCTokenType>$/));
+    before(function() {
+        chakram.addProperty("tctoken", function(respObj) {
+            let xml = respObj.body.replace(/\n/g, "").replace(/\r/g, "").replace(/ /g, "");
+            expect(xml).to.match(/^<TCTokenType><ServerAddress>.*<\/Binding><\/TCTokenType>$/);
         });
     });
 
-    it("returns a tc token", function(done) {
-        frisby.get('https://' + process.env.BOILERPLATE_DOMAIN + '/api/eIdService/getTcTokenUrl', { redirect: 'manual' })
-            .expect('status', 302)
-            .then(function(response) {
-                let location = response.headers._headers.location[0];
-                let tcTokenUrl = location.split("?tcTokenUrl=")[1];
-                return frisby.get(tcTokenUrl)
-                    .expect('status', 200)
-                    .expect('tc token');
-            })
-            .done(done);
+    it("returns a tc token", function() {
+        var response = chakram.get('https://' + process.env.BOILERPLATE_DOMAIN + '/api/eIdService/getTcTokenUrl', { followRedirect: false });
+        expect(response).to.have.status(302);
+        return response.then(function(respObj) {
+            let location = respObj.response.headers.location;
+            let tcTokenUrl = location.split("?tcTokenUrl=")[1];
+            var response = chakram.get(tcTokenUrl);
+            expect(response).to.have.status(200);
+            expect(response).to.be.tctoken;
```

Comment:
I think `return chakram.wait();` should be added here, otherwise the test will pass no matter which status or tctoken is expected.

Reply:
Hey thank you. You're completely right.

Reply:
Resolved!